content/jobs.rst

```diff
@@ -32,10 +36,13 @@ OddBirds:
 - Engage with the software community as part of the job –
   writing articles, sharing tools, and giving talks
 
-We work together to find clients,
-and we work together to build them the best tools we can.
-When there's work to be done, we all do what we can,
-and we get paid for our time.
+Our process keeps design and development tightly integrated --
+everyone working together (with clients and users) towards a unified goal.
+We make decisions as a team,
+find clients together,
+and collaborate to build them the best tools we can.
+We all do what we can,
```

Comment:
I wonder about re-adding the `When there's work to be done,` clause... too negative?